src/App.vue

```diff
@@ -272,7 +272,7 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     if (!this.haveChanges || force) {
       // redirect to dashboard
       const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
-      window.location.assign(dashboardUrl + this.nrNumber)
+      window.location.assign(dashboardUrl + this.getBusinessIdentifier)
```

Comment:
Is "businessIdentified = "tempId" = "temporaryId?
Looking at this diff, I wonder why we have multiple properties for the same thing?

Reply:
done